From: Andy Gospodarek <gospo@redhat.com>
Date: Wed, 27 Oct 2010 19:34:10 -0400
Subject: [net] bnx2: Increase max rx ring size from 1K to 2K
Message-id: <1288208050-18882-3-git-send-email-gospo@redhat.com>
Patchwork-id: 28958
O-Subject: [RHEL5.6 PATCH 2/2] bnx2: Increase max rx ring size from 1K to 2K
Bugzilla: 640026
RH-Acked-by: David S. Miller <davem@redhat.com>
RH-Acked-by: Neil Horman <nhorman@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

This is a backport of upstream commit:

    commit fdb246f526e422b18b48d578085c01ab28ec2c33
    Author: Michael Chan <mchan@broadcom.com>
    Date:   Mon Oct 18 14:30:54 2010 +0000

        bnx2: Increase max rx ring size from 1K to 2K

diff --git a/drivers/net/bnx2.h b/drivers/net/bnx2.h
index 5088560..526d808 100644
--- a/drivers/net/bnx2.h
+++ b/drivers/net/bnx2.h
@@ -6487,8 +6487,8 @@ struct l2_fhdr {
 #define TX_DESC_CNT  (BCM_PAGE_SIZE / sizeof(struct tx_bd))
 #define MAX_TX_DESC_CNT (TX_DESC_CNT - 1)
 
-#define MAX_RX_RINGS	4
-#define MAX_RX_PG_RINGS	16
+#define MAX_RX_RINGS	8
+#define MAX_RX_PG_RINGS	32
 #define RX_DESC_CNT  (BCM_PAGE_SIZE / sizeof(struct rx_bd))
 #define MAX_RX_DESC_CNT (RX_DESC_CNT - 1)
 #define MAX_TOTAL_RX_DESC_CNT (MAX_RX_DESC_CNT * MAX_RX_RINGS)
